src/app/jokes/jokes.component.ts

```diff
@@ -1,15 +1,47 @@
 import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { HttpService } from '../service/http.service';
 
 @Component({
   selector: 'app-jokes',
   templateUrl: './jokes.component.html',
   styleUrls: ['./jokes.component.css']
 })
 export class JokesComponent implements OnInit {
+  jokes: any = [];
```

Comment:
if there is only one joke use singular, if it's an array, use plural... also, create a Joke interface